packages/yoga/src/Avatar/native/Avatar.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { BuildingFilled } from '@gympass/yoga-icons';
+import { string, func, number } from 'prop-types';
+import Box from '../../Box';
+import Image from './Image';
+import Placeholder from './Placeholder';
+
+const Avatar = ({
+  src,
+  fill,
+  placeholder,
+  width = 48,
+  height = 48,
+  borderRadius = 'small',
+  type = 'default',
+}) => {
+  const icon = placeholder || BuildingFilled;
+  return (
+    <Box
+      bgColor="elements.selectionAndIcons"
+      display="flex"
+      width={width}
+      height={height}
+      alignItems="center"
+      justifyContent="center"
+      elevation="small"
+      borderRadius={borderRadius}
+    >
+      {src ? (
+        <Image type={type} source={src} />
+      ) : (
+        <Placeholder fill={fill} icon={icon} />
+      )}
+    </Box>
+  );
+};
+
+Avatar.propTypes = {
+  src: string,
+  placeholder: func,
+  fill: string,
+  width: number,
+  height: number,
+  borderRadius: string,
+  type: string,
+};
+
+Avatar.defaultProps = {
+  src: undefined,
+  placeholder: undefined,
+  fill: undefined,
+  width: undefined,
+  height: undefined,
+  borderRadius: undefined,
+  type: undefined,
+};
+
+Avatar.displayName = 'Avatar';
```

Comment:
Why is this value being assigned? This is the same value that will be inferred by react. I saw that this is done in other places but I didn't understand the motivation.

Reply:
I also don't know the motivation. Can you tell us? @ggdaltoso

Reply:
@leonardokl the @ggdaltoso is on vacation.Is it ok we ask about this when he returnerd?